docs/csharp/programming-guide/generics/constraints-on-type-parameters.md

```diff
@@ -99,11 +97,11 @@ If you uncomment the last line, it won't compile. Both `first` and `test` are de
 
 ## Enum constraints
 
-Beginning in C# 7.3, you can also specify the `System.Enum` type as a base class constraint. This enables type safe programming to cache results from using the static methods in `System.Enum`. The following sample finds all the valid values for an enum type, and then builds a dictionary that maps those values to its string representation.
+Beginning in C# 7.3, you can also specify the <xref:System.Enum?displayProperty=nameWithType> type as a base class constraint. Generics using `System.Enum` provide type safe programming to cache results from using the static methods in `System.Enum`. The following sample finds all the valid values for an enum type, and then builds a dictionary that maps those values to its string representation.
```

Comment:
Missing the CLR remark for the `System.Enum` constraint. Sorry, if I'm too fast, and it comes in the next commit.